dspace/config/dspace.cfg

```diff
@@ -1510,6 +1510,7 @@ log.report.dir = ${dspace.dir}/log
 # _uacct = "UA-XXXXXXX-X"
 # Take this key (just the UA-XXXXXX-X part) and place it here in this parameter.
 # google.analytics.key=UA-XXXXXX-X
+# google.analytics.buffer.limit=256
```

Comment:
Please document (via a comment above this) what 256 means.  Is this 256MB? 256 events? something else?